nifi-nar-bundles/nifi-standard-services/nifi-distributed-cache-services-bundle/nifi-distributed-cache-server/src/main/java/org/apache/nifi/distributed/cache/server/SetCacheServer.java

```diff
@@ -102,4 +100,18 @@ protected void finalize() throws Throwable {
         }
     }
 
+    private byte[] readValue(final DataInputStream dis) throws IOException {
+        final int numBytes = validateInt(dis.readInt(), getMaxReadSize(), "readValue():");
```

Comment:
The `readValue():` parameter appears to be the only value passed to `validateInt()`, so it seems better to remove this argument.
```suggestion
        final int numBytes = validateInt(dis.readInt());
```